en/snap/with-plugins.md

```diff
@@ -420,313 +487,436 @@ Customer receives email notification| Receive notification|  Check MAP menu sett
 </details>
 <hr><br><br>
 
-### Opencart
+### OpenCart
 <hr>
 
-Midtrans ❤️ Opencart! This is official Midtrans extension for the OpenCart E-Commerece platform.
+Midtrans ❤️ OpenCart! This is official Midtrans extension for the OpenCart e-commerece platform.
+
+#### Requirements
+Some of the requirements to continue with the integration process, are listed below.
+* OpenCart minimal 2.0 or later
+* PHP version 5.4 or later
+* MySQL version 5.0 or later
+* Midtrans plugin for OpenCart
+
+#### Installation of OpenCart plugin
+To install OpenCart plugin, follow the steps given below.
+1. Download midtrans OpenCart plugin and extract the zip file.
+    * [OpenCart v3.0](https://github.com/Midtrans/Midtrans-Opencart3)
+    * [OpenCart v2.3](https://github.com/Midtrans/SNAP-Opencart-2.3/)
+    * [OpenCart v2.0, v2.1, v2.2](https://github.com/veritrans/SNAP-Opencart)
```

Comment:
We want AptDocs to help with inconsistencies within our docs, for example, this part is inconsistent because it does not have ZIP download url, you should please add those to make it consistent with other CMS instructions.

e.g: ZIP download url is `https://github.com/Midtrans/Midtrans-Opencart3/archive/master.zip` for `https://github.com/Midtrans/Midtrans-Opencart3`

Reply:
I will include the links wherever required.